### ISSUE TITLE
Docs: Running admin should be in /admin-ui and not /admin

### ISSUE THREAD

**Issue by TKGZ** [amplication/amplication#1390]
The generated folder is called `admin-ui` but the docs still say to run in `/admin`.

**Comment by yuval-hazaz** [comment]
You are correct. The folder name was changed in the last version released last week, so we need to update the docs accordingly.
Thanks for reporting!
